testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -923,8 +925,7 @@ def test_persistent_offsets_readonly(self, tmpdir):
         shutil.rmtree(tmpdir)
 
     def test_offset_lock_created(self):
-        assert os.path.exists(XDR.offsets_filename(self.filename,
-                                                   ending='lock'))
+        assert os.path.exists(XDR.offsets_filename(self.filename))
```

Comment:
This test does not make sense to me. We should be checking that the file is locked in some way. How does one do that with filelock?

Reply:
Please address or explain.

Reply:
> The original test aimed to determine whether a `.lock` file was created so the current test doesn’t seem to make sense to me either.
> 
> The original/current tests don’t actually verify that the file is locked or, more specifically, that the `read_offsets` process waits for the file to be unlocked before proceeding.
> 
> I can’t think of a clever way to mock this behavior. One possible approach could be to spawn a file lock in a separate process for 10 seconds and then verify that the read_offsets function takes more than 10 seconds to complete, indicating it’s waiting for the lock to be released.

True, yes I would the try to see if it could work like that :thinking: